main.cpp

```diff
@@ -181,6 +201,19 @@ void AirConditioner::decreaseHumidity(short targetRelativeHumidity)
         std::cout << "Warning. Remote is being used at max distance" << std::endl;
 }
 
+void AirConditioner::decreaseHumidity(short targetRelativeHumidity, short steps)
+{
+    if (steps <= (100 - targetRelativeHumidity) % 10)
+    {
+        for (short i = 0; i < steps; ++i)
+            {
+                short humidity = (100 - targetRelativeHumidity) - 10 * i;
+                std::cout << "Current humidity is " << humidity << std::endl;
+            }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above
- [ ] revise